### ISSUE TITLE
3.0.x-dev uses addViolationAt in UniqueObject Validator

### ISSUE THREAD

**Issue by ghost** [propelorm/PropelBundle#411]
Hey,

I'm using the 3.0.x-dev branch from PropelBundle for symfony3 and I'm trying to use the unique object validator. However, the validator uses 

```
$this->context->addViolationAt(
    $constraint->errorPath,
    $constraint->message,
    array(
        '{{ object_class }}' => $class,
        '{{ fields }}' => implode($constraint->messageFieldSeparator, $fieldParts)
    )
);
```

which is an undefined method. When submiting the form:

```
{
    "code": 500,
    "message": "Attempted to call an undefined method named \"addViolationAt\" of class \"Symfony\\Component\\Validator\\Context\\ExecutionContext\".\nDid you mean to call \"addViolation\"?",
    "errors": null
}
```

**Comment by ghost** [comment]
Would it possible, that you merge this in 1.5 as well? I get deprecation notices when running my unit tests. Symfony (2.8 now)

**Comment by marcj** [comment]
Well, for that we need to ask @havvg  :)

**Comment by ghost** [comment]
You guys really should consider about developing this bundle forwards. It gives so much power to developers. Much more then doctrine does.

**Comment by havvg** [comment]
I picked it into 1.5, but there are other issues caused by the YAML changes. No guarantee, but it's likely I will get this done today :)

https://travis-ci.org/havvg/PropelBundle/builds/156717109

**Comment by ghost** [comment]
Sounds awesome. Thank you very much.

**Comment by kapowkimc** [comment]
Could this also be merged into 1.6? It seems odd it was merged into 1.5 and 1.6 was left broken.
